dogu/interfaces.go

```diff
@@ -1,17 +1,23 @@
 package dogu
 
 import (
+	"context"
+	"errors"
 	"github.com/cloudogu/cesapp-lib/core"
 )
 
-type SimpleDoguName string
-type DoguNamespace string
+var DoguDescriptorNotFoundError = errors.New("No DoguDescriptor found for that dogu")
+var ConnectionError = errors.New("There are some connection issues")
```

Comment:
🚫 **[checkstyle]** <errname> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>the variable name `ConnectionError` should conform to the `ErrXxx` format
<!-- __reviewdog__:ChAzYWJhNjY3N2Y0ODk5NTFhEgpjaGVja3N0eWxl -->